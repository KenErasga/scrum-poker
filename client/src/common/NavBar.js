import React from 'react'
import { AppBar, Toolbar, Typography } from '@material-ui/core';

const NavBar = ({ isAuthenticated }) => {
    return (
        <div>
            {isAuthenticated ? null :
                <AppBar position="fixed">
                    <Toolbar>
                        <Typography variant="h4" component="h4">
                            Scrum Poker Online - Version 1
        </Typography>
                    </Toolbar>
                </AppBar>}
        </div>
    )
}
export default NavBar;