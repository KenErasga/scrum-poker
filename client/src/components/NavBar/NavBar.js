import React from 'react'
import { AppBar, Toolbar, Typography } from '@material-ui/core';

const NavBar = () => {
    return (
        <div>
            <AppBar position="static">
                <Toolbar>
                    <Typography variant="h4" component="h4">
                        Scrum Poker Online
                    </Typography>
                </Toolbar>
            </AppBar>
        </div>
    )
}
export default NavBar;