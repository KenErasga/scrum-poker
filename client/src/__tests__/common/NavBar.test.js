import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../../common/NavBar';

describe('NavBar', () => {
    it('NavBar renders correctly', () => {
    
    const tree = renderer
        .create(<NavBar />)
        .toJSON();
    expect(tree).toMatchSnapshot();
    });
});
