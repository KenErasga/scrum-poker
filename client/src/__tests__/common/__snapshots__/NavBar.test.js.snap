// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar NavBar renders correctly 1`] = `
<div>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h4
        className="MuiTypography-root MuiTypography-h4"
      >
        Scrum Poker Online
      </h4>
    </div>
  </header>
</div>
`;
